ecdsa/src/sign.rs

```diff
@@ -129,7 +129,7 @@ where
     /// computed using the algorithm described in RFC 6979 (Section 3.2):
     /// <https://tools.ietf.org/html/rfc6979#section-3>
     fn try_sign_digest(&self, digest: D) -> Result<Signature<C>, Error> {
-        let k = rfc6979::generate_k(self.inner.secret_scalar(), digest.clone(), &[]);
+        let k = crate::rfc6979::generate_k(self.inner.secret_scalar(), digest.clone(), &[]);
```

Comment:
We now have both an `rfc6979` module in `ecdsa` as well as a crate with the same name, so we need this disambiguation.

I considered moving that functionality to the new crate under an `ecdsa` feature flag or something, but as mentioned earlier I don't think that's possible without creating a circular dependency between the two crates.

Reply:
That's fine for now. I can look into refactoring later.